help-beta/astro.config.mjs

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+// https://astro.build/config
+export default defineConfig({
+	base: 'help-beta',
+	integrations: [
+		starlight({
+			title: 'Zulip Help Center',
```

Comment:
This string one is user-facing, so should be "Zulip help center".

Reply:
Fixed!